aws/resolve_customer/README.rst

```diff
@@ -1,7 +1,80 @@
-AWS ResolveCustomer interface
-=============================
+AWS ResolveCustomer API
+=======================
 
-The lambda function is expected to be triggerd by an AWS API Gateway.
+The resolve customer interface provides customer and entitelement
+information such that the SUSE Customer Center (SCC) can create
```

Comment:
This is not really SCC specific. Any other application could also call the API provided.

Reply:
I rephrased it